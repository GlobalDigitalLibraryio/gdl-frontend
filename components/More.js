// @flow
/**
 * Part of GDL gdl-frontend.
 * Copyright (C) 2017 GDL
 *
 * See LICENSE
 */

import styled from 'react-emotion';
import media from '../style/media';

/**
 * Right aligned more links in header tags
 *
 */
const More = styled.a`
  float: right;
  font-size: 12px;
  ${media.tablet`
    font-size: 14px;
  `};
  padding: 0 0 5px 5px;
`;
// add a tiny bit of padding to inclease the click surface on mobile (left and bottom only so it doesn't affect layout)

export default More;
