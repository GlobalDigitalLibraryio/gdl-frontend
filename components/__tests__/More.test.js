// @flow
/**
 * Part of GDL gdl-frontend.
 * Copyright (C) 2017 GDL
 *
 * See LICENSE
 */

import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import { createSerializer } from 'jest-emotion';
import * as emotion from 'emotion';
import More from '../More';

expect.addSnapshotSerializer(createSerializer(emotion));

test('Renders', () => {
  const tree = shallow(<More>More</More>);

  expect(toJson(tree)).toMatchSnapshot();
});
