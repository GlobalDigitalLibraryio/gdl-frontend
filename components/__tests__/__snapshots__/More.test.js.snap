// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders 1`] = `
.emotion-0 {
  float: right;
  font-size: 12px;
  padding: 0 0 5px 5px;
}

@media (min-width:768px) {
  .emotion-0 {
    font-size: 14px;
  }
}

<a
  className="emotion-0"
>
  More
</a>
`;
